### ISSUE TITLE
content: ARIA [2020-08-01]

### ISSUE THREAD

**Issue by kaycebasques** [GoogleChrome/web.dev#3324]
* [draft](https://docs.google.com/document/d/14suhqQWR1ZgVmh4JBve9vVpPSG9rMeOYhWTjU5d4bmw/edit)
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/dN7o9wJfK84)
* next step: aaron to find hero image and @robdodson to fix a11y issues

**Comment by robdodson** [comment]
@kaycebasques the a11y issues were fixed a while back, this should be good to go.
